.github/workflows/actions.yaml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest']
+        opa: ['v0.21.0']
+    name: ${{ matrix.os }} opa ${{ matrix.opa }}
+    steps:
+    - uses: actions/checkout@v2
+    - run:
+        sudo curl -L -o /usr/local/bin/opa https://github.com/open-policy-agent/opa/releases/download/{{ matrix.opa }}/opa_linux_amd64
```

Comment:
you need to change to `${{ matrix.opa }}` and this will not work in macOS since this is a linux binary

Reply:
thx, fixed